imagery/i.signatures/i.signatures.html

```diff
@@ -0,0 +1,43 @@
+<h2>DESCRIPTION</h2>
+
+<em>i.signatures</em> module allows to manage signature files generated
+by <a href="i.gensig.html">i.gensig</a>,
+<a href="i.gensigset.html">i.gensigset</a> and
+<a href="i.svm.train.html">i.svm.train</a>.
+The module can perform multiple actions per run. The order of execution
+is "copy", "remove", "rename". If no actions are specified, then the module
+lists (prints) all signature files of specified type.
+
+<h2>EXAMPLES</h2>
+
+Print names of all signature files:
+<div class="code"><pre>
+i.signatures
```

Comment:
Historically and in terms of consistency, we have issues with calls to tools without parameters. Should it start GUI or do something else? I suggest `i.signatures -p` for this.

Reply:
I changed to use a flag for printing